### PR TITLE
Hide sync icon for smart inventory rows in Inventory List

### PR DESCRIPTION
**Pull request by nixocio**
Hide sync icon for smart inventory rows in Inventory List

See: https://github.com/ansible/awx/issues/7681

### CODE REVIEW

File: awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx [hunk 1]
```diff
@@ -101,7 +101,9 @@ function InventoryListItem({
         <DataListItemCells
           dataListCells={[
             <DataListCell key="sync-status" isIcon>
-              <SyncStatusIndicator status={syncStatus} />
+              {inventory.kind !== 'smart' ? (
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx]: I think using the logical && operator would be better here. More concise and a bit easier to ready. so this line would be `inventory.kind !=='smart' &&  <SyncStatusIndicator status={syncStatus} />`

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
Look good. Just 1 change.

File: awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx (hunk 1 above)
Comment (nixocio) [reply awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx]: I will update.

**Review by mabashian — APPROVED** [review]

**Comment by unlikelyzero** [comment]
@nixocio please just make sure that the visual tests are updated to reflect this layout change :)

**Review by AlexSCorey — APPROVED** [review]
Looks good to me.

**Review by tiagodread — APPROVED** [review]

**Pull request merged**